CI test fails: AssertionError: Cannot use images with flatten_messages_as_text=True

CI test fails: https://github.com/huggingface/smolagents/actions/runs/13994614877/job/39186538691
```
FAILED tests/test_models.py::TestModel::test_transformers_message_vl_no_tool - AssertionError: Cannot use images with flatten_messages_as_text=True
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -64,7 +64,7 @@ telemetry = [
 ]
 transformers = [
   "accelerate",
-  "transformers>=4.0.0",
+  "transformers>=4.0.0,<4.50.0",
   "smolagents[torch]",
 ]
 vision = [

